code/ai/aicode.cpp

```diff
@@ -15758,6 +15766,8 @@ void ai_ship_hit(object *objp_ship, object *hit_objp, vec3d *hit_normal)
 	case AIM_WARP_OUT:
 		return;
 		break;
+	case AIM_LUA:
+		return;
 	default:
 		Int3();	//	Bogus mode!
```

Comment:
Since you're in the area, can you replace these INT3's with some calls to UNREACHABLE.